script.js

```diff
@@ -0,0 +1,125 @@
+let quizData;
+let quizNum = 1;
+let answerNum = 0;
+const quizAllNum = 10;
+
+const startQuiz = () => {
+
+	fetch('https://opentdb.com/api.php?amount=10')
+	.then(function(response) {

```

Comment:
ここも、他の場所と同じようにアロー関数でよいと思います。
```suggestion
	.then((response) => {
```

Reply:
承知しました。修正いたします。